src/VirtoCommerce.ReturnModule.Core/Services/IReturnService.cs

```diff
@@ -1,11 +1,15 @@
+using System.Collections.Generic;
 using System.Threading.Tasks;
+using VirtoCommerce.OrdersModule.Core.Model;
 using VirtoCommerce.Platform.Core.GenericCrud;
 using VirtoCommerce.ReturnModule.Core.Models;
 
 namespace VirtoCommerce.ReturnModule.Core.Services
 {
     public interface IReturnService : ICrudService<Return>
     {
-        Task SaveChangesAsync(Return[] returns);
+        Task<IEnumerable<string>> SaveChangesAsync(Return[] returns);
```

Comment:
It would be better if we would pass IEnumerable in parameters and return IList or Array in the result.